app/controllers/users_controller.rb

```diff
@@ -8,6 +9,14 @@ def show
     @chef = User.find(params[:id])
```

Comment:
This line is repeated twice, once here and once in the destroy. Can you create a set_chef method to keep the code dry? thanks

Reply:
yes sure. will do that. thanks for the correction!